code/script.js

```diff
@@ -1,51 +1,215 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat');
-
+const userInput = document.getElementById('userInput');
+const sendBtn = document.getElementById ('sendBtn'); 
+const inputWrapper = document.getElementById ('inputWrapper'); 
+const userForm = document.getElementById ('userForm') ; 
 // If you need any global variables that you can use across different functions, declare them here:
 
+let lineNumber = 1; 
+
+const botReply = (msg) => {
+  showMessage (msg, 'bot') ; 
+}
+
+const userReply = (msg) => {
+  showMessage (msg, 'user') ; 
+}
```

Comment:
Nice variables so you don't need to add both parameters each time you invoke the showMessage function.